napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -267,15 +456,19 @@ def _count_visible(self) -> int:
 
         return count - hidden
 
-    @Slot(PackageMetadata)
+    @Slot(npe2.PackageMetadata)
     def addItem(
         self,
-        project_info: PackageMetadata,
+        project_info_versions: Tuple[
+            npe2.PackageMetadata, List[str], List[str]
+        ],
```

Comment:
There is the problem with types. 
In slot, there is information about `PackageMetadata`, but function annotation suggests `Tuple[npe2.PackageMetadata, List[str], List[str]]`. Could I ask for the agreement?

Reply:
I think I resolved this.  It would only work if I put `tuple` in the slot.  Is that sufficient?

Reply:
Yes